# generated by patch-package 6.5.0 on 2022-03-31 20:04:34
#
# command:
#   npx patch-package @emotion/server --exclude nothing
#
# declared package:
#   @emotion/server: 11.4.0
#
diff --git a/node_modules/@emotion/server/package.json b/node_modules/@emotion/server/package.json
index fdb6ec6..26c61fd 100644
--- a/node_modules/@emotion/server/package.json
+++ b/node_modules/@emotion/server/package.json
@@ -50,9 +50,6 @@
   "bugs": {
     "url": "https://github.com/emotion-js/emotion/issues"
   },
-  "browser": {
-    "./dist/emotion-server.cjs.js": "./dist/emotion-server.browser.cjs.js"
-  },
   "preconstruct": {
     "entrypoints": [
       "./index.js",
